src/components/book/styles/book.css

```diff
@@ -8,18 +8,6 @@
   gap: 16px;
 }
 
```

Comment:
### STYLES APPLICABLE TO NAV

- Hey, please, kindly update your styles to match the template design as much as you can and add the missing avatar image to improve your design. See the image below:


*template*
![Screenshot from 2022-12-16 00-18-02](https://user-images.githubusercontent.com/59397159/207987448-56c192ae-cd88-48ad-b93f-d0b8ea13aacf.png)

*your design*
![Screenshot from 2022-12-16 00-16-45](https://user-images.githubusercontent.com/59397159/207987452-0e1e5ec7-9661-443d-98a6-24f6be81aad2.png)